src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx

```diff
@@ -256,7 +256,7 @@ In order to debug this project, add an executable project to this solution which
     <value>Failed to migrate XProj project {0}. Could not find project.json at {1}.</value>
   </data>
   <data name="EditProjectFileCommand" xml:space="preserve">
-    <value>Edit {0}.{1}</value>
+    <value>Edit {0}{1}</value>
```

Comment:
What's the point of splitting the extension from the file name? Why isn't this just Path.GetFileName on the project file name?